backend/alexa_tutorial.md

```diff
@@ -0,0 +1,71 @@
+##Steps to building an alexa skill:
+
+###Prerequisites:
+In order to develop an Alexa skill you'll need access to 2 accounts.
+
+1. **aws.amazon.com**: An IAM user with access to the webservices@intrepid.io account
+2. **developer.amazon.com**:  webservices@intrepid.io for amazon developers, this login info is in LastPass.
```

Comment:
Sherpa articles are ideally consumed by any user.  Could you either phrase this in a way that is less Intrepid specific, or remove the references to our specific accounts?

Or maybe Sherpa has evolved since and is really just a launching point for us.